.gitignore

```diff
@@ -102,4 +102,4 @@ gas-report.txt
 
 contracts/test/fuzzing/crytic-export
 
-on-demand-request.json
\ No newline at end of file
+Functions-request.json
```

Comment:
nit: Not a huge fan of uppercase filenames but not a big deal.

Reply:
We can address later if necessary